### DISCUSSION TITLE
Where can I see the list of all the badges available?

### DISCUSSION THREAD

**Discussion by taliba570** [community/community#29322] [Other Feature Feedback, Questions, & Ideas] [answered]
I was looking through different sections but I was not able to find the possible achievements list, can someone help me find that?

**Comment by melvincwng** [comment, marked as answer]
@taliba570 , you can refer to this link here: https://github.com/Schweinepriester/github-profile-achievements for a full list of achievements.

Hope this clarifies 😄
